pkg/sdk/dynamic_table_test.go

```diff
@@ -0,0 +1,159 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDynamicTableCreate(t *testing.T) {
```

Comment:
We try to put more validation tests on unit test level, consult e.g. `database_role_test.go`.

Reply:
done